/entrance_exploration_action_server:
  ros__parameters:
    osm_file_path: '/workspaces/light-map-navigation/src/llm_exploration_py/OSM/medium.osm'
    exploration_radius: 2.0
    exploration_points: 5
    camera_topic: '/camera_sensor/image_raw'
    map_frame: 'map'
    bk_frame: 'base_link'
    transform_matrix: [1.0, 0.0, -500000.0, 0.0, 1.0, -4483000.0, 0.0, 0.0, 1.0]
    navigation_feedback_interval: 2.0
    service_timeout: 1.0 